java/ql/src/Security/CWE/CWE-297/InsecureJavaMail.qhelp

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+  <overview>
+    <p>JavaMail is commonly used in Java applications to send emails. There are popular third-party libraries like Apache Commons Email which are built on JavaMail and facilitate integration. Authenticated mail sessions require user credentials and mail sessions can require SSL/TLS authentication. It is a common security vulnerability that host-specific certificate data is not validated or is incorrectly validated. Failing to validate the certificate makes the SSL session susceptible to a man-in-the-middle attack.</p>
+    <p>This query checks whether SSL certificate is validated when username/password is sent in authenticator and when SSL is enabled.</p>
```

Comment:
Word missing?
"...is sent in the authenticator"
"...is sent to the authenticator"
Not sure which one is correct here.

Reply:
Thanks for pointing this out! I rephrased it a bit in a89009f, let me know if something can be improved.